src/cc/frontends/p4/compiler/ebpfAction.py

```diff
@@ -60,24 +70,27 @@ def __init__(self, p4action, program):
 
         self.builtin = False
         self.invalid = False  # a leaf action which is never
-                              # called from a table can be invalid.
+        # called from a table can be invalid.
```

Comment:
This should either be ignored or the comment should be moved atop the variable assignment.